[docs] Reduce the use of "our", "We"... (#17347)
mui-org/material-ui: docs/src/pages/discover-more/related-projects/related-projects.md

# Related projects

> <p class="description">A carefully curated ist of libraries that are worth having a look at.</p>

Author: ```suggestion
<p class="description">A carefully curated list of libraries that are worth having a look at.</p>
```

Because the scope of problems Material-UI solves is limited, we try to play nicely with
other libraries.

> Feel free to submit a pull request to add more projects; we will accept them if they match the criteria.

Author: ```suggestion
Feel free to submit a pull request to add another project; it will be accepted if it is suitable.
```
